doc/nrf/releases/release-notes-1.7.0.rst

```diff
@@ -0,0 +1,589 @@
+.. _ncs_release_notes_170:
+
+|NCS| v1.7.0 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the TF-M, MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
+nRF53 Series devices are now supported for production.
+Wireless protocol stacks and libraries may indicate support for development or support for production for different series of devices based on verification and certification status.
+See the release notes and documentation for those protocols and libraries for further information.
+
+Highlights
+**********
+
+* Software support for :ref:`zephyr:thingy53_nrf5340`:
+
+  * :ref:`nrf_machine_learning_app` application with Edge Impulse ML development platform, supported for development.
+  * :ref:`Matter weather station <matter_weather_station_app>` application
+  * Bluetooth® samples:
+
+    * :ref:`central_uart`
+    * :ref:`peripheral_lbs`
+    * :ref:`peripheral_uart`
+
+  * Bluetooth mesh samples:
+
+    * :ref:`bluetooth_mesh_light`
+    * :ref:`bluetooth_mesh_light_switch`
+
+* Multi-image DFU for nRF53 Series with simultaneous application and network core firmware update.
+* :ref:`ug_radio_fem` for the nRF21540 FEM range extender for product development.
+
+  * :ref:`direct_test_mode` and :ref:`radio_test` sample support.
+  * Application support for protocols based on Bluetooth® LE and 802.15.4 (Thread and Zigbee), using :ref:`nrfxlib:mpsl`.
+  * Support for :ref:`nRF21540 development kit (DK) <nrf21540dk_nrf52840>`, :ref:`nRF21540 evaluation kit (EK) <ug_radio_fem_nrf21540_ek>` and custom board configurations.
+
+* Added Wi-Fi Coex feature for the 802.15.4 radio driver, supported for development.
+* Added support for NFC and *pair before use* type of accessories to the Apple Find My add-on.
+* nRF Connect for Cloud is now nRF Cloud.
+* Updated the required :ref:`minimum CMake version <gs_recommended_versions>` to 3.20.0.
+
```

Comment:
Add this bullet: "Added support for nRF Connect SDK development in nRF Connect for Visual Studio Code"